src/Umbraco.Web.UI.Client/src/packages/multi-url-picker/link-picker-modal/link-picker-modal.element.ts

```diff
@@ -126,10 +160,29 @@ export class UmbLinkPickerModalElement extends UmbModalBaseElement<UmbLinkPicker
 			name: this.value.link.name || name,
 			type: unique ? type : undefined,
 			unique,
-			url,
+			url: url ?? this.value.link.url,
 		};
 
 		this.#partialUpdateLink(link);
+		this.#validationContext.validate();
```

Comment:
Do we need to await the result here or is it all right if the validation result lands later?

```suggestion
		await this.#validationContext.validate();
```